src/components/cursor-controller.js

```diff
@@ -26,6 +25,9 @@ AFRAME.registerComponent("cursor-controller", {
   init: function() {
     this.inVR = false;
     this.isMobile = AFRAME.utils.device.isMobile();
+    if (this.isMobile) {
+      this._setCursorVisibility(false);
```

Comment:
Won't this mean you won't see your cursor on mobile vr devices until your first click on something?